src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,11 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(stringNumbers -> Stream.of(stringNumbers.split(",")))
+                .mapToInt(Integer::parseInt).filter(number -> isEven(number)).min()
```

Comment:
every opeartion form new line 
```suggestion
                .mapToInt(Integer::parseInt)
                .filter(number -> isEven(number))
                .min()
```